### ISSUE TITLE
ParameterTypeHintSniff does not recognize list<T> as traversable

### ISSUE THREAD

**Issue by janedbal** [slevomat/coding-standard#1487]
Following method is not reported as missing native `array` typehint for `$items`.
```php
    /**
     * @param list<string> $items
     */
    public function myMethod($items): void {}
```

PHPStan added proper list support in [1.9.0](https://github.com/phpstan/phpstan/releases/tag/1.9.0), do you plan to implement this within slevomat/cs too? Thanks

**Comment by kukulich** [comment]
It makes sense. It just needs new options in all three sniffs.

**Comment by kukulich** [comment]
Implemented in https://github.com/slevomat/coding-standard/commit/54d5575ac1fdf9fef9fae2101c444a6d681489ce

**Comment by janedbal** [comment]
Thank you
